app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.table_name = "actors"
+        self._connection = sqlite3.connect("cinema")

```

Comment:
You forgot to add a file extension, for example `cinema.db3`

Reply:
Thanks for your review! Done all changes.